cms/static/coffee/spec/main.coffee

```diff
@@ -180,10 +178,13 @@ requirejs.config({
             exports: "sinon"
         },
         "jasmine-imagediff": {},
-        "jasmine-stealth": {
+        "common/js/spec_helpers/jasmine-extensions": {
+            deps: ["jquery"]
+        },
+        "common/js/spec_helpers/jasmine-stealth": {
             deps: ["underscore", "underscore.string"]
         },
-        "jasmine-waituntil": {
+        "common/js/spec_helpers/jasmine-waituntil": {
```

Comment:
This one isn't in npm, but there is a similar sounding library: https://www.npmjs.com/package/jasmine-waits-for-callback